src/parser/python/index.ts

```diff
@@ -0,0 +1,56 @@
+import { Program } from 'estree'
+
+import { parsePythonToEstreeAst } from '../../py-slang/src'
+import { Chapter, Context } from '../../types'
+import { FatalSyntaxError } from '../errors'
+import { AcornOptions, Parser } from '../types'
+import { positionToSourceLocation } from '../utils'
+
+export class PythonParser implements Parser<AcornOptions> {
+  private chapter: Chapter
+  constructor(chapter: Chapter) {
+    this.chapter = chapter
+  }
+  parse(
+    programStr: string,
+    context: Context,
+    options?: Partial<AcornOptions>,
+    throwOnError?: boolean
+  ): Program | null {
+    try {
+      // parse the scheme code
```

Comment:
```suggestion
      // parse the python code
```
To avoid confusion with Scheme side.